### ISSUE TITLE
 Xvfb startup problems

### ISSUE THREAD

**Issue by jensens** [plone/jenkins.plone.org#195]
Sometimes (but not always) jenkins dies with Xvfb startup problems, see also http://jenkins.plone.org/job/plone-5.1-python-2.7-robot/1233/console

I can not make a clear conclusion when it happens, but seems if more than one job on a node is running (but not always as it seems).

```
TCP port allocation complete
Xvfb starting$ Xvfb :1 -screen 0 1024x768x24 -fbdir /home/jenkins/xvfb-1233-2398987545882624513.fbdir
(EE) 
Fatal server error:
(EE) Server is already active for display 1
	If this server is no longer running, remove /tmp/.X1-lock
	and start again.
(EE) 
unlink: No such file or directory
unlink  failed, Invalid argument
ERROR: Xvfb failed to start, consult the lines above for errors
```

**Comment by tisto** [comment]
@jensens the xvfb plugin in Jenkins makes sure screens are properly allocated. Sometimes one screen is blocked by a process that died unexpected. Re-starting the node or killing that process solves the problem. Idealy we would start/stop VMs for each test run, but we are not there yet.

**Comment by gforcada** [comment]
Some days ago I saw two nodes in jenkins that were pointing to the same server, which caused lots of problems like the one you describe...

**Comment by jensens** [comment]
@gforcada @tisto at the moment the problem is again immanent.

**Comment by gforcada** [comment]
@jensens if you at least point us to the specific jobs and/or nodes that are failing it would be a bit easier to solve... :-)

Seems it's on Node1 right? I will turn it off for a while. Anyway is the old nodes that should be put off eventually... probably not now though.

**Comment by gforcada** [comment]
I restarted http://jenkins.plone.org/job/plone-5.1-python-2.7-robot/ let's see

**Comment by gforcada** [comment]
@jensens thanks for reporting!

**Comment by gforcada** [comment]
We will be able to fix this once we spin a vm/container to run the tests. Until then, please keep reporting issues as you see them coming.
